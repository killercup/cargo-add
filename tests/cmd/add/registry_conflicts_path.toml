bin.name = "cargo-add"
args = ["add", "my-package", "--registry", "alternative", "--path", "/path/here"]
status.code = 2
fs.sandbox = true

[env.add]
CARGO_IS_TEST="1"
