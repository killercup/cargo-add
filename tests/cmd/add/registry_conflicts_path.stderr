error: The argument '--registry <REGISTRY>' cannot be used with '--path <PATH>'

USAGE:
    cargo add --registry <REGISTRY> --upgrade <METHOD> <CRATE>...

For more information try --help
